pykrige/core.py

```diff
@@ -54,6 +58,26 @@
 from scipy.optimize import minimize
 
 
+def great_circle_distance(lon1, lat1, lon2, lat2):
+    # Calculate dot product of both vectors:
```

Comment:
Yes, I was refering to the euclidean dot product of both vectors. Will clear that up in the doc string!